debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MySqlSourceTypeInSchemaIT.java

```diff
@@ -252,4 +252,30 @@ public void shouldPropagateSourceTypeByDatatype() throws SQLException, Interrupt
         assertThat(f2SchemaParameters).includes(
                 entry(TYPE_NAME_PARAMETER_KEY, "FLOAT"), entry(TYPE_LENGTH_PARAMETER_KEY, "8"), entry(TYPE_SCALE_PARAMETER_KEY, "4"));
     }
+
+    /**
+     * The MySQL database is constructed using random database names and sometimes those names may actually
+     * align so that the column source filters match the database name rather than the actual columns.
+     * This method is to help preface the desired columns with the qualified table name to avoid this collision.
+     *
+     * @param tableName the unqualified table name
+     * @param columns the individual regexs for columns that propagate source type information
+     * @return the configuration value string
+     */
+    private String getColumnPropagateSourceType(String tableName, String... columns) {
+        if (columns.length == 0) {
+            return null;
+        }
+
+        final StringBuilder result = new StringBuilder();
+        for (int i = 0; i < columns.length; ++i) {
+            final String column = columns[i];
+            result.append(DATABASE.qualifiedTableName(tableName) + "." + column);
+            if ((i + 1) < columns.length) {
+                result.append(",");
+            }
+        }
+        System.out.println(result.toString());
```

Comment:
I think this should be removed

Reply:
Oh, I thought I had removed that before pushing.  Great catch.